app/assets/javascripts/scp-heatmap.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Render Morpheus heatmap
+ */
+function renderMorpheus(
```

Comment:
Always love to see refactoring like this.

Reply:
Regarding function name: I agree `renderMorpheus` isn't ideal, especially now that we have another Morpheus graphic in dot plots.  I'm thinking `renderHeatmap`, consistent with how we omit vendor from other new visualization rendering function names (e.g. `renderScatterPlot` instead of `renderPlotlyScatterPlot`).

Regarding `console.log`: I also lean towards omitting them all.  Given the different perspectives, I'll keep this as is for now.  I've added it as a topic for this Thursday's [engineering strategy](https://docs.google.com/document/d/11eNlwMjwKoUcx4TQflzxw4DuN9q03fNNLphUuAVNIc4/edit#heading=h.4zti8yqhognv) meeting, so we can hopefully reach clear consensus.

Reply:
e0bf891 refines function names.